server/api/ballot_manifest.py

```diff
@@ -115,6 +119,36 @@ def process():
 
     process_file(session, file, process)
 
+    # If CVR file already uploaded, try reprocessing it, since it depends on
+    # batch names from the manifest
+    if jurisdiction.cvr_file:
```

Comment:
should we always reprocess the CVR file, or only if the batch set has changed? Or maybe we also need to if the numbers have changed?

I suppose it won't kill us to reprocess, so I can live with this solution if it's simpler.

Reply:
Yeah we could be smarter about it, but I don't see a huge advantage. Right now I'm using the dumb approach for all the file reprocessing.